test/scales/colorScaleTests.ts

```diff
@@ -0,0 +1,129 @@
+///<reference path="../testReference.ts" />
+
+describe("Scales", () => {
+  describe("Color Scale", () => {
+
+    describe("Basic Usage", () => {
+      let defaultColors = [
+        "#5279c7", "#fd373e", "#63c261", "#fad419", "#2c2b6f",
+        "#ff7939", "#db2e65", "#99ce50", "#962565", "#06cccc"
+      ];
+
+      it("default colors are generated", () => {
+        let scale = new Plottable.Scales.Color();
```

Comment:
not sure what's the best practice here. it seems like you call this same line first time in all the tests, is it worth to create a beforeEach? (this is a question rather than comment)